Amazon.QLDB.Driver/session/QldbSession.cs

```diff
@@ -111,9 +111,11 @@ public T Execute<T>(Func<TransactionExecutor, T> func)
             ValidationUtils.AssertNotNull(func, "func");
 
             ITransaction transaction = null;
+            string transactionId = null;
```

Comment:
Did you check if this is consumed anywhere? It must be used for something, even if it's just a printed stacktrace.

What does that stacktrace look like if it's null? Would an empty string be better here?

Otherwise LGTM

Reply:
Here is the stacktrace using the transacton ID:
```
      A recoverable exception has occurred. Attempting retry 1. Errored Transaction ID: (null).
Amazon.QLDBSession.Model.InvalidSessionException: E
   at Amazon.QLDB.Driver.QldbSession.StartTransaction() in /Users/byronlin/projects/github/amazon-qldb-driver-dotnet/Amazon.QLDB.Driver/session/QldbSession.cs:line 164
   at Amazon.QLDB.Driver.QldbSession.Execute[T](Func`2 func) in /Users/byronlin/projects/github/amazon-qldb-driver-dotnet/Amazon.QLDB.Driver/session/QldbSession.cs:line 117
```

Reply:
Updated default transaction ID to None,

Now the log stacktrace looks like:
```
      A recoverable exception has occurred. Attempting retry 2. Errored Transaction ID: None.
Amazon.QLDBSession.Model.InvalidSessionException: E
   at Amazon.QLDB.Driver.QldbSession.StartTransaction() in /Users/byronlin/projects/github/amazon-qldb-driver-dotnet/Amazon.QLDB.Driver/session/QldbSession.cs:line 164
   at Amazon.QLDB.Driver.QldbSession.Execute[T](Func`2 func) in /Users/byronlin/projects/github/amazon-qldb-driver-dotnet/Amazon.QLDB.Driver/session/QldbSession.cs:line 117
```